docs/shared/nx-core.md

```diff
@@ -172,27 +215,24 @@ that adding a `require()` creates a dependency and that some dependencies cannot
 
 Running `yarn nx run-many --target=test --all` will test all projects in parallel.
 
-Running `yarn nx run-many --target=build --projects=app1,app2` will build `proj1` and `proj2` and their
-dependencies in parallel. Note that if `app1` depends on the output of its dependency (e.g., `shared-components`), Nx
-will build `shared-components` first and only then will build the app.
+// placeholder (should we have people make these projects so they can see more in the dep-graph and commands won't fail?)
+Running `yarn nx run-many --target=build --projects=app1,app2` builds `proj1` and `proj2` and their
+dependencies in parallel. Note that if `app1` depends on the output of its dependency (e.g., a `shared-components` library), Nx
+will build the `shared-components` library first and only then build the app.
 
 ### Nx Knows What Is Affected
 
 Running `yarn nx affected --target=test` will test all the projects affected by the current PR.
 
 ### Nx Caches and Distributes Tasks
 
-Running `yarn nx build app1` will cache the file artifacts and the terminal output, so if you run it again the command
-will execute instantly because the results will be retrieved from cache. If you use `Nx Cloud` the cache will be shared
-between you, your teammates, and the CI agents. Nx can also distribute tasks across multiple machines while preserving
-the developer experience of running it on a single machine.
+Running `yarn nx build app1` caches the file artifacts and terminal output. When you run the command again, it will execute instantly because the results are retrieved from the cache. If you use [Nx Cloud](https://nx.app/), the cache is shared between you, your teammates, and the CI agents. Nx distribute tasks across multiple machines while preserving the developer experience of running on a single machine.
 
-This works because Nx's computation caching and distributed task execution work on the process level. It doesn't matter
-what `build` means. It can be an npm script, a custom Nx executor, a Gradle task. Nx will handle it in the same way.
+// placeholder: link to what DTE is?
+This works because Nx's computation caching and distributed task execution work on the process level. It doesn't matter what `build` means. It can be an npm script, a custom Nx executor, a Gradle task. Nx will handle it in the same way.
```

Comment:
good time to explain more about DTE and why it's a great feature, probably link out to some explainer page?